### PR TITLE
chore: using `tsx` instead of `ts-node`

### PR DESCRIPTION
**Pull request by Dunqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

`tsx` is simpler, faster, and more timely to maintain!

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      using `tsx` instead of `ts-node`     |
| 🇨🇳 Chinese |      使用 tsx 代替 ts-node     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at ec4f7f5</samp>

Improved TypeScript build performance and cleaned up dependencies. Switched to `tsx` for transpiling TypeScript files in `package.json` scripts.

### <samp>🤖 Generated by Copilot at ec4f7f5</samp>

* Replace `ts-node` with `tsx` for faster TypeScript transpilation in scripts `test`, `lint:style`, `argos`, and `version` ([link](https://github.com/ant-design/ant-design/pull/44688/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L50-R59), [link](https://github.com/ant-design/ant-design/pull/44688/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L80-R80), [link](https://github.com/ant-design/ant-design/pull/44688/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L99-R100))
* Remove unused dependencies `@swc/core`, `@swc/helpers`, and `ts-node` from `package.json` ([link](https://github.com/ant-design/ant-design/pull/44688/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L180-L181), [link](https://github.com/ant-design/ant-design/pull/44688/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L302))

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,16 +47,16 @@
   "scripts": {
     "prepare": "is-ci || husky install",
     "api-collection": "antd-tools run api-collection",
-    "authors": "ts-node --esm --project tsconfig.node.json scripts/generate-authors.ts",
+    "authors": "tsx scripts/generate-authors.ts",
```
Comment (afc163) [inline package.json:50]: tsx 从哪来的，是否要在 package.json 加个包?

**Comment by MadCcc** [comment]
要清理下 ts-node 和 swc 的依赖

File: package.json (hunk 1 above)
Comment (Dunqing) [reply package.json:50]: done

**Comment by yoyo837** [comment]
忘了还有个 [skip-verify-files](https://github.com/ant-design/ant-design/labels/skip-verify-files) 标签

**Comment by Dunqing** [comment]
> 要清理下 ts-node 和 swc 的依赖
清理了

> 另外脚本最好都手动跑一下，有些不会在 ci 里执行但是会用到

测了一遍，没有出现 `tsx` 导致的报错

**Review by MadCcc — APPROVED** [review]

**Comment by Dunqing** [comment]
> 居然如此顺利

好像是去掉 types 跑的，跟跑 js 一样，所以平滑迁移

**Comment by MadCcc** [comment]
> 好像是去掉 types 跑的，跟跑 js 一样，所以平滑迁移

应该是 esbuild 直接做的编译，没有类型检查，tsconfig 是不是还是得覆盖一下？

**Comment by renny-ren** [comment]
坐等这个 merge. 我现在本地`yarn start` 跑不起来，项目 clone 了两次都不行
遇到 `ts-node` 这里就报 `Unknown file extension ".ts"`
image

**Comment by Dunqing** [comment]
> 应该是 esbuild 直接做的编译，没有类型检查，tsconfig 是不是还是得覆盖一下？

又测试了一下，目前 tsconfig.json 配置能检查 scripts 的类型错误，所以 CI 会校验

**Pull request merged**